libkbfs/init.go

```diff
@@ -365,9 +366,25 @@ func Init(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn, onI
 			onInterruptFn()
 		}
 
-		os.Exit(1)
+		close(done)
 	}()
 
+	errCh := make(chan error)
+	go func() {
```

Comment:
This is so that we can select on `done` and `errCh` below, in case SIGINT happens in the middle of the actual init. With this, we can (and I did) remove the `os.Exit(1)` from the SIGINT handler so everything finishes naturally. I'll add a comment. Thanks for pointing it out!